app/build.gradle

```diff
@@ -27,11 +28,11 @@ android {
     signingConfigs {
         playStore
     }
-    compileSdkVersion 24
-    buildToolsVersion '25.0.2'
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 26
```

Comment:
Are we sure this won't have any side effects? Updating the target SDK version disables compatibility behavior for behavior changes in newer Android versions after all. Glancing over the behavior changes document I don't see any obvious breakage for us, but I'd like us to try first nevertheless ;-)